internal/pkg/aws/acm/acm.go

```diff
@@ -66,23 +71,31 @@ func (a *ACM) ValidateCertAliases(aliases []string, certs []string) error {
 	}
 	for _, alias := range aliases {
 		if !validAliases[alias] {
+			for certARN, domains := range AllowedDomainsCert {
+				log.Infoln(fmt.Sprintf("Your imported Certificate '%s' protects the following domains:", certARN))
```

Comment:
I think we should avoid `log`ging in arbitrary packages. What do you think of making the returned error a custom type and putting these in its `RecommendedActions`?